pkg/login/social/social.go

```diff
@@ -235,6 +222,25 @@ func NewOAuthService() {
 	}
 }
 
+func createOrganizationMapping(authConfig *auth.AuthConfig) map[string][]SocialGroup {
+	var orgMap = make(map[string][]SocialGroup)
+	for _, auth := range authConfig.AuthMappings {
+		groups := auth.Groups
```

Comment:
nit: could simply use `range auth.Groups` in loop

Reply:
Updated.